docs/source/users/index.md

```diff
@@ -355,10 +356,22 @@ Currently, we offer experimental support for GPT4All. To get started, first
 decide which models you will use. We currently offer three models from GPT4All:
 
 | Model name                   | Model size | Model bin URL                                              |
```

Comment:
Do we need to provide the model size? Does GPT4All have its own page that we can link to? I worry that this info might become outdated easily.

Reply:
Unless the models change, this should not change. If the models will change, they will have different names/versions.